### ISSUE TITLE
Energy Net Hardsuit Module - The deployed net doesn't properly net people if they are moving.

### ISSUE THREAD

**Issue by SharpAurora** [Aurorastation/Aurora.3#5303]
The energy net module which can be purchased by the ninja antagonist does not seem to be working fully properly. 

The net when thrown at the target roots them in place if they are stationary. However if they were mobile, it breaks even if it hits them.

Test 1: The net was thrown when the target was pacing up and down. The target was rooted briefly before it broke without brute force. The target and the deployed net were not on the same tile.

Test 2: Similar to the above, except the target was able to stay fully mobile. The deployed net was not on the same tile as the target, breaking almost instantly.

Test 3: We tested it while the target was stationary. The target and the deployed net were on the same tile. The target was unable to move out.

tl;dr The deployed net from the module has an issue rooting mobile people.

**Comment by Ornias** [comment]
this is still a problem lol
